src/components/App.tsx

```diff
@@ -3981,15 +3656,10 @@ class App extends React.Component<ExcalidrawProps, AppState> {
 
     if (isNaN(pointer.x) || isNaN(pointer.y)) {
       // sometimes the pointer goes off screen
-      return;
     }
-    this.portal.socket &&
-      // do not broadcast when more than 1 pointer since that shows flickering on the other side
-      gesture.pointers.size < 2 &&
-      this.portal.broadcastMouseLocation({
```

Comment:
the mouse pointers are updated based on some core logic like checking `isPanning`, `gesture.pointers.size < 2 ` and also the listeners are on canvas so might be better to call the callback from here so host need not worry about adding the callbacks or these logic but update the name of callback to something better like `onMousePointerUpdate` may be so host can do whatever it wants. 
@dwelle#7968 what do you think ?

Reply:
Right. Although I'm not sure how to do this without coupling the two pretty tightly. E.g. we don't wanna invoke the callback if not collaborating or  if `pointers.size > 1`, but both of these conditions are super specific to excalidraw.com :/.

Reply:
Yeah, I was thinking the same lets pass `gesture.pointers` 👍